bundles/org.openhab.binding.solax/README.md

```diff
@@ -0,0 +1,210 @@
+
+For example: I want to charge my car if the battery of my inverter is 90%+ and my overall consumption is only from the solar power.
+The binding retrieves structured data from the wifi module, parses it and pushes it into the inverter thing where each channel represents a specific information (inverter output power, voltage, PV1 power, etc)
+
+
+Hostname/IP address of the wi-fi module (can be any IP address from internal DHCP wi-fi network) [String]
+The serial number of the Wi-fi module is the default password (not sure if it can be changed) [String]
+Time interval to poll the data from the module [number in seconds]
+Please give it a try and provide me a feedback if it works for you. I will update the releases as soon as I implement some of the backlog.
+## Supported Things
+
+| Thing             | Thing Type | Description                                                                   |
+|-------------------|------------|-------------------------------------------------------------------------------|
+| localConnect      | Bridge     | The bridge is used to communicate directly to the wifi module of the inverter.|
+| inverter          | Thing      | This is model representation of inverter with all the data available as a channels (Ex. inverter output power, voltage, PV1 power, etc) |
+
+## Discovery
+
+Discovery is available. Once the localConnect bridge is configured with the necessary parameters, it can discover automatically the inverter with all it's channels as a separate thing.
+
+## Thing Configuration
+
+### Local Connect Bridge parameters
+
+| Parameter         | Description                            |
+|-------------------|----------------------------------------|
+| refresh           | Value is in seconds. Defines the refresh interval when the binding polls from the inverter's wifi module. Optional parameter. Default 10 seconds. |
+| password          | Password for accessing the Wifi module (the serial number of the wifi). Mandatory parameter.  |
+| hostname          | IP address or hostname of your Wifi module. If hostname is used must be resolvable by OpenHAB. Mandatory parameter.  |
+
+The bridge does not have any channels. It is used only to connect to the module and retrieve the data. The data is shown in the Inverter thing
+
+### Inverter Thing Configuration
+
+### Inverter Output Channels
+
+| Channel                  | Type                       | Description                                                                               |
+|--------------------------|----------------------------|-------------------------------------------------------------------------------------------|
+| inverterOutputPower      | system.electric-power      | The output power of the inverter [W]           |
```

Comment:
Should be item type, not channel type:
```suggestion
| inverterOutputPower      | Number:Power               | The output power of the inverter [W]           |
```

Please fix all.

Reply:
Can you give me a hint where to see these?

Reply:
You should be able to see them in the UI, but you could also cheat by looking here 😄
https://github.com/openhab/openhab-core/blob/main/bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/DefaultSystemChannelTypeProvider.java

Reply:
Do you know if there is something like Number:Percentage for the battery level?

Reply:
I see that humidity is Number:Dimensionless (and humidity is also %)